pcdsdevices/qadc.py

```diff
@@ -0,0 +1,122 @@
+from ophyd import Component as Cpt
+from ophyd import Device, EpicsSignal
+
+from .interface import BaseInterface
+from .variety import set_metadata
+
+
+class QadcCommon(BaseInterface, Device):
+    """
+    Common class for Abaco FMC digitizers. Used in Qadc and Qadc134.
+    """
+
+    ch0 = Cpt(EpicsSignal, ":CH0", kind="normal")
+    ch1 = Cpt(EpicsSignal, ":CH1", kind="normal")
+    ch2 = Cpt(EpicsSignal, ":CH2", kind="normal")
+    ch3 = Cpt(EpicsSignal, ":CH3", kind="normal")
+    config = Cpt(EpicsSignal, ":CONFIG", kind="config")
+    start = Cpt(EpicsSignal, ":START", kind="normal")
+
+
+class QadcLcls1Timing(BaseInterface, Device):
+    """
+    LCLS-I timing interface for FMC134 QADCs.
+    """
+    trig_event = Cpt(
+        EpicsSignal, ":TRIG_EVENT_RBV", write_pv=":TRIG_EVENT", kind="config"
+    )
+
+
+class QadcLcls2Timing(BaseInterface, Device):
+    """
+    LCLS-II timing interface for FMC134 QADCs.
+    """
+    trig_rate_mode = Cpt(EpicsSignal, ":TRIG_RATEMODE", kind="config")
+    trig_fixed_rate = Cpt(EpicsSignal, ":TRIG_FIXEDRATE", kind="config")
+    trig_ac_rate = Cpt(EpicsSignal, ":TRIG_ACRATE", kind="config")
+    trig_ts_mask = Cpt(EpicsSignal, ":TRIG_TSMASK", kind="config")
+    trig_seq_num = Cpt(EpicsSignal, ":TRIG_SEQNUM", kind="config")
+    trig_seq_bit = Cpt(EpicsSignal, ":TRIG_SEQBIT", kind="config")
+    trig_partition = Cpt(EpicsSignal, ":TRIG_PARTITION", kind="config")
+
+
+class Qadc134Common(QadcCommon):
+    """
+    Common class for FMC134 digitizers.
+    """
+    trig_delay = Cpt(
+        EpicsSignal, ":TRIG_DELAY_RBV", write_pv=":TRIG_DELAY", kind="config"
+    )
+    full_en = Cpt(EpicsSignal, ":FULL_EN_RBV", write_pv=":FULL_EN", kind="config")
```

Comment:
Any chance you could add some `doc=` strings to describe what all these components are (for this entire file, really)? It may be obvious to you, but it's definitely not obvious to me (and if I had to guess, the rest of the team as well).